pkg/apis/planetscale/v2/defaults.go

```diff
@@ -88,7 +88,7 @@ const (
 	// DefaultMysqlPortName is the name for the MySQL port.
 	DefaultMysqlPortName = "mysql"
 
-	defaultVitessLiteImage = "us.gcr.io/planetscale-vitess/lite:2020-01-14.6b55d845"
+	defaultVitessLiteImage = "us.gcr.io/planetscale-vitess/lite:2020-03-07.f569401b"
```

Comment:
Is this needed for this PR or are you just updating it to keep our image up to date? Since this may affect other things, I'd include it in the PR description for posterity's sake.

Reply:
This is needed for this PR. That is the first vitess version that has the azblob backup engine